plugin-server/src/worker/vm/extensions/posthog.ts

```diff
@@ -32,12 +32,16 @@ export function createPosthog(server: Hub, pluginConfig: PluginConfig): DummyPos
             if (!server.kafkaProducer) {
                 throw new Error('kafkaProducer not configured!')
             }
-            // ignore the promise, run in the background just like with celery
+
+            const partitionKeyHash = crypto.createHash('sha256')
+            partitionKeyHash.update(`${data.team_id}:${data.distinct_id}`)
+            const partitionKey = partitionKeyHash.digest('hex')
```

Comment:
@macobo 

![Uploading Screenshot 2022-03-18 at 10.49.59.png…]()

Reply:
perfect, thank you!